### ISSUE TITLE
Higher test loss when using model with pretrained weights

### ISSUE THREAD

**Issue by gongzix** [MedARC-AI/fMRI-reconstruction-NSD#46]
Hello, thank you for your great work.
I meet some problems when training MindEye. I downloaded the pretrained model for subject1, and found the reconstruction performance is good as expected. 
However, when I attempted to train a new model with Train_MindEye.py, I found the reconstuction results all poor. Especially, when using model with pretrained weights, the test loss is much higher than model with random Initialization. and it seem to be wrong when the optimizers load_state_dict of your public pretrained weights.
So i wanna to know whether there are some errors in training script or rather, I didn't do it right. 
Thanks again.

**Comment by xqoby** [comment]
I meet the same problem, may I ask you how to solve it?

**Comment by gongzix** [comment]
After changing from half precision to full precision, I obtained the correct result. I hope it can be helpful to you.

**Comment by Phoenix-BM** [comment]
Hello, I also met this problem.
My training records are listed as follows:

![Image](https://github.com/user-attachments/assets/55b01a51-9f36-4520-8154-1f585ecadd94)

As shown in the table, when I train a new model from scratch for subject 1, following the training script, the results are poorer than those with pre-trained weights. I also changed the half-precision to full precision, but there seems to be little improvement. Do you have any other suggestions?

Thanks a lot!

**Comment by Phoenix-BM** [comment]
Ohhh, I solved this problem by setting num_devices to 1.
